src/documents/NPSP_Documents/NPSP_Data_Import_Template_csv.csv

```diff
@@ -1 +1 @@
NPSP__CONTACT1_SALUTATION__C,NPSP__CONTACT1_FIRSTNAME__C,NPSP__CONTACT1_LASTNAME__C,NPSP__CONTACT1_BIRTHDATE__C,NPSP__CONTACT1_TITLE__C,NPSP__CONTACT1_PERSONAL_EMAIL__C,NPSP__CONTACT1_WORK_EMAIL__C,NPSP__CONTACT1_ALTERNATE_EMAIL__C,NPSP__CONTACT1_PREFERRED_EMAIL__C,NPSP__CONTACT1_HOME_PHONE__C,NPSP__CONTACT1_WORK_PHONE__C,NPSP__CONTACT1_MOBILE_PHONE__C,NPSP__CONTACT1_OTHER_PHONE__C,NPSP__CONTACT1_PREFERRED_PHONE__C,NPSP__CONTACT2_SALUTATION__C,NPSP__CONTACT2_FIRSTNAME__C,NPSP__CONTACT2_LASTNAME__C,NPSP__CONTACT2_BIRTHDATE__C,NPSP__CONTACT2_TITLE__C,NPSP__CONTACT2_PERSONAL_EMAIL__C,NPSP__CONTACT2_WORK_EMAIL__C,NPSP__CONTACT2_ALTERNATE_EMAIL__C,NPSP__CONTACT2_PREFERRED_EMAIL__C,NPSP__CONTACT2_HOME_PHONE__C,NPSP__CONTACT2_WORK_PHONE__C,NPSP__CONTACT2_MOBILE_PHONE__C,NPSP__CONTACT2_OTHER_PHONE__C,NPSP__CONTACT2_PREFERRED_PHONE__C,NPSP__ACCOUNT1_NAME__C,NPSP__ACCOUNT1_STREET__C,NPSP__ACCOUNT1_CITY__C,NPSP__ACCOUNT1_STATE_PROVINCE__C,NPSP__ACCOUNT1_ZIP_POSTAL_CODE__C,NPSP__ACCOUNT1_COUNTRY__C,NPSP__ACCOUNT1_PHONE__C,NPSP__ACCOUNT1_WEBSITE__C,NPSP__ACCOUNT2_NAME__C,NPSP__ACCOUNT2_STREET__C,NPSP__ACCOUNT2_CITY__C,NPSP__ACCOUNT2_STATE_PROVINCE__C,NPSP__ACCOUNT2_ZIP_POSTAL_CODE__C,NPSP__ACCOUNT2_COUNTRY__C,NPSP__ACCOUNT2_PHONE__C,NPSP__ACCOUNT2_WEBSITE__C,NPSP__HOME_STREET__C,NPSP__HOME_CITY__C,NPSP__HOME_STATE_PROVINCE__C,NPSP__HOME_ZIP_POSTAL_CODE__C,NPSP__HOME_COUNTRY__C,NPSP__DONATION_DONOR__C,NPSP__DONATION_AMOUNT__C,NPSP__DONATION_DATE__C,NPSP__DONATION_NAME__C,NPSP__DONATION_RECORD_TYPE_NAME__C,NPSP__DONATION_STAGE__C,NPSP__DONATION_TYPE__C,NPSP__DONATION_DESCRIPTION__C,NPSP__DONATION_MEMBER_LEVEL__C,NPSP__DONATION_MEMBERSHIP_START_DATE__C,NPSP__DONATION_MEMBERSHIP_END_DATE__C,NPSP__DONATION_MEMBERSHIP_ORIGIN__C,NPSP__DONATION_CAMPAIGN_NAME__C,NPSP__PAYMENT_CHECK_REFERENCE_NUMBER__C,NPSP__PAYMENT_METHOD__C
```

Comment:
I don't think you added the new campaign member status to the template.  I'd include it just after the Campaign Name field.